includes/wikia/models/DesignSystemFooterModel.class.php

```diff
@@ -0,0 +1,428 @@
+				'key' => 'wikia-create-wiki-description'
+			],
+			'links' => [
+				[
+					'type' => 'link-text',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'wikia-create-wiki-link-start-wikia'
+					],
+					'href' => 'http://www.wikia.com/Special:CreateNewWiki'
+				]
+			]
+		],
+		'community_apps' => [
+			'header' => [
+				'type' => 'line-text',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'wikia-community-apps-header'
+				]
+			],
+			'description' => [
+				'type' => 'translatable-text',
+				'key' => 'wikia-community-apps-description'
+			],
+			'links' => [
+				[
+					'type' => 'link-image',
+					'image' => 'company/store-appstore',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'wikia-community-apps-link-app-store'
+					],
+					'href' => 'https://itunes.apple.com/pl/developer/wikia-inc./id422467077'
+				],
+				[
+					'type' => 'link-image',
+					'image' => 'company/store-googleplay',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'wikia-community-apps-link-google-play'
+					],
+					'href' => 'https://play.google.com/store/apps/developer?id=Wikia,+Inc.'
+				]
+			]
+		],
+		'advertise' => [
+			'header' => [
+				'type' => 'line-text',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'wikia-advertise-header'
+				]
+			],
+			'links' => [
+				[
+					'type' => 'link-text',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'wikia-advertise-link-media-kit'
+					],
+					'href' => 'http://www.wikia.com/mediakit'
+				],
+				[
+					'type' => 'link-text',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'wikia-advertise-link-contact'
+					],
+					'href' => 'http://www.wikia.com/mediakit#contacts'
+				]
+			]
+		],
+	];
+
+	private $allowedVerticals = [
+		'games',
+		'movies',
+		'tv',
+	];
+
+	private $wikiId;
+
+	public function __construct( $wikiId ) {
+		parent::__construct();
+
+		$this->wikiId = $wikiId;
+	}
+
+	public function getData() {
+		$this->setLicensingAndVertical();
+
+		return $this->data;
+	}
+
+	private function setLicensingAndVertical() {
+		$params = [
+			'license' => $this->getLicenseData(),
+		];
+		$verticalData = $this->getVerticalData();
+
+		if ( !empty( $verticalData ) ) {
+			$params['vertical'] = $verticalData;
+		}
+
+		$this->data['licensing_and_vertical'] = [
+			'description' => [
+				'type' => 'translatable-text',
+				'key' => 'wikia-licensing-and-vertical-description',
+				'params' => $params
+			],
+		];
+	}
+
+	private function getVerticalData() {
+		$wikiVerticalData = WikiFactoryHub::getInstance()->getWikiVertical( $this->wikiId );
+		$vertical = null;
+		if ( $wikiVerticalData['id'] ) {
+			$vertical = $wikiVerticalData['short'];
+			$link = 'http://fandom.wikia.com/' . $vertical;
+		}
+
+		if ( is_null( $vertical ) ) {
+			return [];
+		}
+
+		$data = [
+			'type' => 'link-text',
```

Comment:
This won't pass the validation (we should fix the schema if it does). When there is no link the type should be `line-text`.